### ISSUE TITLE
Drifts 21: availability of barometer data

### ISSUE THREAD

**Issue by DavorJ** [DOV-Vlaanderen/groundwater-logger-validation#62]
Availability of barometer data for 12h intervals:

![image](https://user-images.githubusercontent.com/4161918/94372928-0fb4c680-0102-11eb-879c-26a97142f34b.png)

The high variance in function of timestamp is because in this period there are lots of barometers that operate on 24h basis instead of 12h basis:
![image](https://user-images.githubusercontent.com/4161918/94372961-4e4a8100-0102-11eb-877b-5bd11ee4a221.png)
That is why the curve constantly jumps between 60 and 90 barometers.

Plotting in function of 24h gives less variance:
![image](https://user-images.githubusercontent.com/4161918/94372930-15aaa780-0102-11eb-8795-32d7a376a32a.png)

And here is how zoomed-in 2019-2020 looks like:
![image](https://user-images.githubusercontent.com/4161918/94373012-97023a00-0102-11eb-8f45-541510cdb457.png)

Mid 2019 there is data from some 100 barometers, and then it drops to 50 starting in 2020. This gradual drop (or addition) in barometers might cause inconsistencies in multi comparison (cf. #51).

**Comment by fredericpiesschaert** [comment]
@DavorJ the drop from mid 2019 is just data that weren't available yet when we made the test series, but I guess you figured that out :)
Another item that might influence the multi comparison are the data from the BAOL5xx series. There are some serious doubts about the functioning of these baro's, so I would suggest to drop them for further analysis.

**Comment by DavorJ** [comment]
> @DavorJ the drop from mid 2019 is just data that weren't available yet when we made the test series, but I guess you figured that out :)

Yes :). Point is that we have to take into consideration potentially missing reference data.
